### DISCUSSION TITLE
Open some URL in client side after click on connection side

### DISCUSSION THREAD

**Discussion by LucasFelipeMacedo** [cedrozor/myrtille#322] [General]
Hello Cedrozor, how you doing?

Thanks for your work, this tool is extraordinary, it's helping us a lot.

Is it possible to make myrtille open a URL in the client's browser from within the connection?

For example:

https:\\google.com <--- Connection side URL

user clicks

[Execute some function]

and https:\\google.com <-- Opens client side

**Comment by cedrozor** [comment]
Hi, sorry I just noticed your post. It could be possible, but how determine if the clicked url should be opened server or client side? The simplest is to copy it then paste it client side, into a new browser tab.
